ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -162,49 +166,50 @@ class LeftNavMenuViewController: UIViewController {
+            guard let strongSelf = self else {
                 return
             }
-            menuAction()
+            strongSelf.menuAction?()
         }
 
-        var statusCellChildren: [MSFListCellState] = []
+        let menuSection = leftNavMenuList.state.createSection()
+
+        let statusCellState = menuSection.createCell()
+
+        statusCellState.title = LeftNavPresence.available.cellTitle
+        statusCellState.backgroundColor = .systemBackground
+        let statusImageView = LeftNavPresence.available.imageView
+        statusCellState.leadingUIView = statusImageView
 
         for presence in LeftNavPresence.allCases {
-            let statusCellChild = MSFListCellState()
+            let statusCellChild = statusCellState.createChildCell()
             statusCellChild.leadingViewSize = .small
-            statusCellChild.title = presence.cellTitle()
-            statusCellChild.leadingUIView = presence.imageView()
+            statusCellChild.title = presence.cellTitle
+            statusCellChild.leadingUIView = presence.imageView
             statusCellChild.backgroundColor = .systemBackground
-            statusCellChild.onTapAction = {
-                self.setPresence(presence: presence)
+            statusCellChild.onTapAction = { [weak self] in
+                guard let strongSelf = self else {
+                    return
+                }
+                strongSelf.setPresence(presence: presence)
             }
-
-            statusCellChildren.append(statusCellChild)
         }
 
-        let resetStatusCell = MSFListCellState()
+        let resetStatusCell = statusCellState.createChildCell()
         resetStatusCell.title = "Reset status"
         resetStatusCell.leadingViewSize = .small
         resetStatusCell.backgroundColor = .systemBackground
         let resetStatusImageView = UIImageView(image: UIImage(named: "ic_fluent_arrow_sync_24_regular"))
         resetStatusImageView.tintColor = FluentUIThemeManager.S.Colors.Foreground.neutral4
         resetStatusCell.leadingUIView = resetStatusImageView
-        resetStatusCell.onTapAction = {
-            self.setPresence(presence: .available)
+        resetStatusCell.onTapAction = { [weak self] in
+            guard let strongSelf = self else {
+                return
+            }
+            strongSelf.setPresence(presence: .available)
         }
-        statusCellChildren.append(resetStatusCell)
-
-        let menuSection = leftNavMenuList.state.createSection()
-
-        statusCell = menuSection.createCell()
-        statusCell.title = LeftNavPresence.available.cellTitle()
-        statusCell.backgroundColor = .systemBackground
-        let statusImageView = LeftNavPresence.available.imageView()
-        statusImageView.tintColor = FluentUIThemeManager.S.Colors.Presence.available
-        statusCell.leadingUIView = statusImageView
-        statusCell.children = statusCellChildren
+        statusCell = statusCellState
```

Comment:
nit: do you mind moving this to right after the initialization of statusCellState? (below line 178)